HangMan/src/hangman.rs

```diff
@@ -0,0 +1,139 @@
+use std::vec;
+use dioxus::prelude::*;
+
+//component for managing the image of the hangman
+#[inline_props]
+pub fn Hangman(cx:Scope, stage: usize, reset_image: bool) -> Element {
+    
+    let css_status = use_ref(cx, || {vec![true;11]});    
+
+    //changes the hidden status of the next image in queque when the user makes a guess
+    use_memo(cx, stage, |stage| {
```

Comment:
`use_memo` should only be used when you care about the return value. If you don't care about the return value, then you can use the `use_effect` hook instead.